python/dgl/graphbolt/impl/ondisk_dataset.py

```diff
@@ -979,6 +979,16 @@ class BuiltinDataset(OnDiskDataset):
         .. note::
             Reverse edges are added to the original graph.
 
+    **igb-hom-[tiny|small|medium]**
+        The igb-hom-[tiny|small|medium] dataset is a homogeneous citation network,
+        which is designed for developers to train and evaluate GNN models with 
+        high fidelity. See more details in `igb-hom-[tiny|small|medium] 
+        <https://mlcommons.org/2024/06/gnn-for-mlperf-training-v4/>`_.
```

Comment:
The reference should be to the github repository of IGB datasets.

Reply:
I see. i will change the URL.